atd-vzv/src/views/summary/PeopleByDemographics.js

```diff
@@ -275,6 +275,7 @@ const PeopleByDemographics = () => {
       <Row>
         <Col>
           <HorizontalBar
+            redraw
```

Comment:
🤔

Reply:
Since we talked about this in dev sync, leaving this here for future reference. https://github.com/jerairrest/react-chartjs-2/blob/3855af1f3bc5ec57199bc5af6aa7d61577e82206/src/index.js#L81